scrapers/nus-v2/package.json

```
@@ -20,6 +20,7 @@
     "@types/bunyan": "1.8.6",
     "@types/deep-diff": "1.0.0",
     "@types/fs-extra": "9.0.6",
+    "@types/he": "^1.1.1",
```

Needed for Renovate bot 

```suggestion
    "@types/he": "1.1.1",
```

---

fixed in 0bee168